### DISCUSSION TITLE
How can I define a domain in _config.yml to use in links

### DISCUSSION THREAD

**Discussion by DavidThielen** [community/community#72750] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi all;

Is there a way to define a value in _config.yml like:

```
appdomain: https://myapp.com
```
And then in index.md have links like: `go to [login]({appdomain}/account/login) to gain access.`

thanks - dave

**Comment by bartzalewski** [comment, marked as answer]
```
# _config.yml
appdomain: https://myapp.com
```

```
# index.md
go to [login]({{ site.appdomain }}/account/login) to gain access.
```

**Reply by DavidThielen** [reply]
thank you

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
